recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/wayland/weston/camera_front.rules

```diff
@@ -0,0 +1 @@
+SUBSYSTEM=="video4linux", ENV{ID_BUS}=="usb", ENV{ID_V4L_CAPABILITIES}=="*capture*", SYMLINK+="cam_front"
```

Comment:
May be it's better to name file camera_usb.rules, not camera_front.rules.
And same for symlink - maybe just 'camera'.
On one side we have to take into account other types of camera. But also we can make assumption that only one type is present in system. So need to discuss.

Reply:
Good point. I was thinking about it. If you ok, we will certainly rename it accordingly in the follow up series when adding support for other camera types. Currently in the context of restoring pv camera support task, I propose to keep it unchanged.